webpack.config.js

```diff
@@ -1,52 +1,38 @@
 const
   webpack = require('webpack'),
-  path = require('path'),
   version = require('./package.json').version;
 
 module.exports = {
+  name: 'browser',
   mode: 'production',
   entry: './index.js',
   output: {
     path: `${__dirname}/dist`,
     filename: 'kuzzle.js',
-    library: 'KuzzleSDK',
+    library: 'kuzzle-sdk',
     libraryTarget: 'umd'
```

Comment:
BTW, why this?

Reply:
I asked @Njuelle about this, as I don't know a thing about how webpacked modules work in browsers. :astonished: